docs/spec/confirm.zh-CN.md

```
@@ -16,16 +16,22 @@ title: 确认
 
 #### 显性确认
 
+<ImagePreview>
+<img class="preview-img no-padding" src="https://mdn.alipayobjects.com/huamei_iwk9zp/afts/img/A*E6wURJNuKgUAAAAAAAAAAAAADgCCAQ/fmt.webp">
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)